Gemfile.lock

```diff
@@ -508,6 +508,7 @@ GEM
       google-cloud-core (~> 1.2)
       googleauth (~> 0.9)
       mini_mime (~> 1.0)
+    google-protobuf (3.17.0)
```

Comment:
Was there a change to the Gemfile that brought these in?  Just checking that everything got committed that should be.